ddev/ddev
# feat: enable corepack for yarn/pnpm version detection, fixes #5987

## The Issue

DDEV ships with Yarn 1 installed globally, but Drupal 11 uses Yarn 4.

## How This PR Solves The Issue

[Corepack](https://nodejs.org/api/corepack.html) is enabled instead, which adds Yarn 1 by default but will download specific versions of Yarn if specified in package.json.

## Manual Testing Instructions

Ensure Yarn 1 is still available by default:

```
$ yarn --version
1.22.22
```

Start a project on Drupal 11 and ensure Yarn 4 works:
```
$ cd core
$ yarn --version
4.1.1
```

## Automated Testing Overview



## Related Issue Link(s)

#5987

## Release/Deployment Notes
